Makefile

```diff
@@ -0,0 +1,2 @@
+all:
+	docker-compose build --build-arg USER_ID=$(shell id -u) --build-arg GROUP_ID=$(shell id -g)
```

Comment:
i removed the last argument `ansible-obs` because it was leading to create a container named `ansible-obs_ansible-obs` which is not what we want.